modules/steam.nix

```diff
@@ -409,5 +438,71 @@ in
         WINE_CPU_TOPOLOGY = "8:0,1,2,3,4,5,6,7";
       };
     })
+    (mkIf cfg.autoStart {
+      assertions = [
+        {
+          assertion = !config.systemd.services.display-manager.enable;
+          message = "No Display Managers must be enabled when jovian.steam.autoStart is used";
+        }
+      ];
+
+      services.xserver = {
+        enable = true;
+        displayManager.lightdm.enable = false;
+        displayManager.startx.enable = true;
```

Comment:
In a pure wayland setup, startx here wouldn't be needed right?

Reply:
Right, it isn't needed and can be set to `lib.mkForce false` locally.